src/lib/components/BucketAggregation/BucketAggregationValues.js

```diff
@@ -97,11 +100,13 @@ BucketAggregationValues.propTypes = {
   }),
   onFilterClicked: PropTypes.func.isRequired,
   overridableId: PropTypes.string,
+  parentValue: PropTypes.string,
 };
 
 BucketAggregationValues.defaultProps = {
   childAgg: null,
   overridableId: "",
+  parentValue: "",
```

Comment:
Isn't it this required now? If yes, default props should appear only for optional arguments:
```suggestion

```

Reply:
We are going for a different solution, where we don't specify any more the id of the element. There is no need to pass the parent anymore